pyproject.toml

```diff
@@ -16,7 +16,7 @@
 
 [tool.poetry]
 name = "pytest-splunk-addon"
-version = "4.0.0-beta.4"
+version = "3.0.8"
```

Comment:
Why do we need this change?
It will be reverted by the bot anyway after you merge this PR?

Reply:
Not really. That's the configuration for the release to update pyproject.toml file after it does the release.

Check out https://github.com/splunk/pytest-splunk-addon/commit/e09f44b932fbde1d6e3e0d5deb33f307b761d8f7 and then bot updates the release branch with this commit https://github.com/splunk/pytest-splunk-addon/commit/93eb600b01335d8f6b7753d11e088d044808df85.

Reply:
I guess you wanted to have a release after this one: https://github.com/splunk/pytest-splunk-addon/commit/1cdf1a35acf50c0c5e28c73ba12b3b17953ea316

Reply:
That release did not work because one of tests was broken (https://github.com/splunk/pytest-splunk-addon/runs/8119930281?check_suite_focus=true).
And release requires both of those checks to pass: https://github.com/splunk/pytest-splunk-addon/blob/50b681bc1d4320d7e811a014525f006708c38c87/.github/workflows/build-test-release.yml#L233-L234

Reply:
So, you can just push "chore" commit with the poetry change and it will trigger a release.

Reply:
This change is needed since we have this: https://github.com/splunk/pytest-splunk-addon/blob/50b681bc1d4320d7e811a014525f006708c38c87/.releaserc#L45 I went through it already. Poetry is another issue